docs/realm.js

```diff
@@ -472,6 +472,7 @@ class Realm {
  * @typedef Realm~PropertyType
  * @type {("bool"|"int"|"float"|"double"|"string"|"decimal128"|"objectId"|"date"|"data"|"list"|"linkingObjects"|"<ObjectType>")}
  *
+ * @property {Mixed} "mixed" - Property value that allow any of the following types (`"bool","int","float","double","string","decimal128","objectId","date","data"`), this type is nullable by default.
```

Comment:
Mixed can also hold objects, not only primitives.

Reply:
Yes that's correct but that's something we plan to support in the next release.